map/models.py

```diff
@@ -18,4 +20,4 @@ def __unicode__(self):
 
     @property
     def picture_url(self):
-        return "/static/map/" + self.picture.url
+        return self.picture.url
```

Comment:
Im wondering how this will integrate with the work done for the search functionality?

Reply:
That is a really good point and we should discuss about it. 
1. So ideally I think it would be nice if we have a Google Map UI index page
2. After we search, a bunch of marker display. 
3. While the user click on the marker, the image display, and other information display with the marker.